src/smc_sagews/smc_sagews/sage_server.py

```diff
@@ -1038,6 +1038,8 @@ def execute(self, code, namespace=None, preparse=True, locals=None):
                             # BUGFIX: be careful to *NOT* assign to _!!  see https://github.com/sagemathinc/cocalc/issues/1107
                             block2 = "sage.misc.session.state_at_init = dict(globals());sage.misc.session._dummy=sage.misc.session.show_identifiers();\n"
                             exec compile(block2, '', 'single') in namespace, locals
+                            b2a = "try: load(os.environ['SAGE_STARTUP_FILE'])\nexcept: pass\n"
```

Comment:
If init.sage has an exception for any reason, e.g., something goes wrong when it is loading, will this code you wrote make that error be silently hidden?   E.g., if init.sage contains `1/0`, then Sage should show an error while starting up.

I will merge this as is, assuming it works, but please open a followup issue about silently ignoring errors in init.sage.